### PR TITLE
Magnus changes for XR

### PR DESCRIPTION
**Pull request by fluffy**

### CODE REVIEW

**Comment by fluffy** [comment]
I think the text in https://github.com/rtcweb-wg/jsep/pull/530 is a better way to go than this text (thought closely related).

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -5856,6 +5856,12 @@ a=msid:-
           deliver a copy of the RTCP packet to the "m=" line
           associated with that SSRC.</t>
 
+          <t> If the packet is fo type XR, as defined in RFC3611, it
```
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: fo -> of

Also, this needs a reference and a section #.

File: draft-ietf-rtcweb-jsep.xml [hunk 2]
```diff
@@ -5856,6 +5856,12 @@ a=msid:-
...
+          depends on the report. A large number of these report
```
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: What depends on the report?

**Comment by aboba** [comment]
+1 to fluffy/ekr.

**Pull request closed without merging**